src/components/vidle.ts

```diff
@@ -111,6 +111,7 @@ const Vidle = Vue.extend({
       this.counter = window.setInterval(this.countdown, 1000)
     },
     clearTimer() {
+      this.$emit("clearTimer")
```

Comment:
The event name is not descriptive from my perspective, can you rename it to "refresh"?

Reply:
Done!